vividus-plugin-rest-api/src/main/resources/vividus-plugin/spring.xml

```diff
@@ -7,7 +7,7 @@
         http://www.springframework.org/schema/util https://www.springframework.org/schema/util/spring-util.xsd"
     default-lazy-init="true">
 
-    <bean class="org.vividus.http.validation.ResourceValidator" >
+    <bean class="org.vividus.http.validation.ResourceValidator" scope="prototype">
```

Comment:
that's the easiest solution, but in future it may lead to issue if one resource validator is needed to be injected in 2 places, so it's better:
- to declare a separate bean of HTTP client for ResourceValidator: e.g. `resourceValidatorHttpClient`
- inject 2 beans of `ResourceValidator` in 2 different places (and reuse HTTP client bean)